userdoc.md

```diff
@@ -10,6 +10,6 @@
    * [Tiny Tiny RSS](/app_ttrss_en)
    * [Transmission](/app_transmission_en)
    * [Radicale](/app_radicale)
-   * [RoundCube](/app_roundcube_en)
```

Comment:
Why did you remove Roundcube link?
[The page exist](https://yunohost.org/#/app_roundcube).